### PR TITLE
Refactored Organization detail view to show all events by the org, and added display of event privacy & publish state.

### PR DESCRIPTION
**Pull request by melinath**
Privacy is now displayed on list and detail pages for events; publish state is displayed on list view only, and only if user is an admin for the event.

The idea here was to prep for event duplication by making sure that organizers have a way to view *all* the events they might want to duplicate. I also think this display makes more sense for an org detail page generally.

### CODE REVIEW

**Comment by melinath** [comment]
This at least partially addresses #458 as well.

File: brambling/templates/brambling/_event_list_item.html [hunk 1]
```diff
@@ -13,7 +13,18 @@ <h5 class="datecard-month">{{ event.start_date|date:"M" }}</h5>
 			</div>
 		</div>
 	</a>
-	<h3>{{ event.name }}</h3>
+	<h3>
+		{{ event.name }}
+		{% if is_admin and not event.is_published %}
+			<small class='text-danger tipped' title="You can see this event because you're are an admin for it">
```
Comment (harrislapiroff) [inline brambling/templates/brambling/_event_list_item.html]: I'm gonna change from danger to something else. Danger should only be for things that you can fix or really should not do.

Comment (harrislapiroff) [inline brambling/templates/brambling/_event_list_item.html]: Actually. Lemme see how this looks.

File: brambling/templates/brambling/event/order/shop.html [hunk 2]
```diff
@@ -179,32 +179,23 @@ <h4 id="myLargeModalLabel">{{ options.grouper.name }}</h4>
...
+									{% include "brambling/event/_privacy.html" %}
 								</div>
-								<div class="media text-large">
-									<div class="media-left">
-										<i class="fa fa-map-marker fa-fw"></i>
-									</div>
-									<div class="media-body">
-										{% include "brambling/event/_where.html" %}
-									</div>
+								<div class='margin-trailer-half'>
```
Comment (harrislapiroff) [inline brambling/templates/brambling/event/order/shop.html]: This whole bit is gonna require some visual/responsive review in a sec.

File: brambling/templates/brambling/_event_list_item.html (hunk 1 above)
Comment (melinath) [inline brambling/templates/brambling/_event_list_item.html]: Someday later this should provide better instruction for how to publish.

**Pull request merged**